### PR TITLE
Luxeradio portfolio- branding section

### PR DESCRIPTION
**Pull request by cp-priyanka-g**

### CODE REVIEW

File: vue-frontend/src/components/portfolio/BrandingSection.vue [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    </div>
+  </section>
+</template>
+<script>
+import AspectRatio from "@/components/utils/AspectRatio.vue";
+export default {
+  props: ["response"],
+  data() {
+    return {
+      flex1: this.response.details.slice(0, 3),
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/portfolio/BrandingSection.vue:80]: This section is common so slice() is not appropriate

File: vue-frontend/src/luxeradio-data.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import branding1 from "@/assets/images/portfolio/luxeradio/branding/luxeradio_br
 import branding2 from "@/assets/images/portfolio/luxeradio/branding/luxeradio_branding_2.png";
 import branding3 from "@/assets/images/portfolio/luxeradio/branding/luxeradio_branding_3.png";
 import branding4 from "@/assets/images/portfolio/luxeradio/branding/luxeradio_branding_4.png";
-import branding5 from "@/assets/images/portfolio/luxeradio/branding/luxeradio_branding_5.png";
+import brandVideo from "@/assets/images/portfolio/luxeradio/branding/GIF.mp4";
```
Comment (cp-shruti-s) [inline vue-frontend/src/luxeradio-data.js:8]: Give the proper name of video file

**Comment by cp-shruti-s** [comment]
No need to change the name of the image file, you can modify luxeradio json data

File: vue-frontend/src/components/portfolio/BrandingSection.vue [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  @apply tw-py-8  sm:tw-px-32 sm:tw-py-8 md:tw-px-14 xl:tw-p-16  2xl:tw-px-32 tw-bg-white tw-text-center;
+}
+.image-gap {
+  @apply tw-mt-12;
+}
+img {
+  @apply tw-w-full tw-h-full tw-object-cover;
+}
+.video-border {
+  @apply tw-rounded-b-[90px] tw-border-b-8 tw-border-x-8 tw-border-solid tw-border-white tw-px-8 tw-pb-8;
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/BrandingSection.vue:103]: As tailwind have concept of inlin ecss, We should add all classes inline. Remove CSS classes from CSS block

File: vue-frontend/src/components/portfolio/BrandingSection.vue [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    >
+      <div class="v2-header-3-text" v-html="response.title"></div>
+    </div>
+  </section>
+  <section class="background-image tw-relative">
+    <aspect-ratio height="56.26%">
+      <img :src="response.backgroundImage" />
+    </aspect-ratio>
+  </section>
+  <section class="tw-relative tw-bg-white">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/BrandingSection.vue:14]: Is this section will have parallax effect or not?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/portfolio/BrandingSection.vue (hunk 3 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/portfolio/BrandingSection.vue:103]: i have used this class to apply border video conditionally , that's why i find this way appropriate.

File: vue-frontend/src/components/portfolio/BrandingSection.vue (hunk 4 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/portfolio/BrandingSection.vue:14]: yes

Comment (cp-sumi-k) [reply vue-frontend/src/components/portfolio/BrandingSection.vue:17]: I am not able to see parallax effect

**Review by cp-shruti-s — APPROVED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**